### ISSUE TITLE
i have a problem in tra_loss and val_loss

### ISSUE THREAD

**Issue by pandaxue** [metal3d/keras-video-generators#26]
sorry,i have a problem in my project,  my project is same to you when i use HMDB database(3classes),,so i want to try my dataset, it is  sign language videos,i select 3classes(150videos) to try, but the tra_loss and val_loss does not decrease, it keeps about 1.1 all the time, i dont how to do decrease loss, I know it's not related to your project, but I thought you might be able to help me,

**Comment by metal3d** [comment]
We're working on french language sign recognition also at Smile (https://smile.eu). The poject is named "Koda" (https://koda-inc.fr)
This is very complex, you need to manipulate video and use SlidingFrameGenerator with a `sequence_time` that corresponds to the mean time of gestures.

You need a lot of videos and a very powerful GPU if you use more thant 8 frames and frame size > 64px.

Unfortunatly, at this time, because the project is in progress, we cannot give more information about the model construction and how we train it.
